dali/c_api/c_api.cc

```diff
@@ -56,7 +56,9 @@ void SetExternalInput(daliPipelineHandle *pipe_handle, const char *name, const v
   data.ShareData(const_cast<void *>(data_ptr), tl_shape.num_elements() * elem_sizeof);
   data.Resize(tl_shape, type_info);
   data.SetLayout(layout);
-  pipeline->SetExternalInput(name, data, stream, flags & DALI_ext_force_sync);
+  pipeline->SetExternalInput(name, data, stream,
+                             flags & DALI_ext_force_sync,
```

Comment:
open question: Should we just pass flags all the way?

Reply:
IMHO, flags should be restricted to `c_api.cc`. In C++ there's a better way of expressing flags than bits in an int ;)